Renaming to baggage (#27)
w3c/baggage: baggage/HTTP_HEADER_FORMAT.md

# Baggage HTTP Header Format

> The baggage header is used to propagate user-supplied key-value pairs through a trace.

Reviewer: Text keeps referring to "trace", which is only tangentially related. It would be useful to introduce a term of "distributed request" or "distributed execution" at the beginning and use that, instead of "trace".

Reviewer 2: I agree with this point. The person reading the spec should not have to be familiar with the term `trace` or the trace context spec.

Author: Let's look through the rest of the wording after the renaming is done. Issue created.

> The baggage header is used to propagate user-supplied key-value pairs through a trace.

Reviewer: ```suggestion
The `Baggage` header is used to propagate user-supplied key-value pairs through a distributed request.
```

A received header MAY be altered to change or add information and it MUST be passed on to all downstream request.

> Multiple baggage headers are allowed. Values can be combined in a single header according to  [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).

Reviewer: ```suggestion
Multiple `Baggage` headers are allowed. Values can be combined in a single header according to  [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
```

*See [rationale document](HTTP_HEADER_FORMAT_RATIONALE.md) for details of decisions made for this format.*

# Header Name

Header name: `Baggage`

@TODO: Agree on the header name

> @TODO: Add more information on casing once we agree on the final header name

Reviewer: remove these TODOs?

Single header:

```

> Bag: userId=sergey,serverNode=DF:28,isProduction=false

Reviewer: ```suggestion
Baggage: userId=sergey,serverNode=DF:28,isProduction=false
```
